### ISSUE TITLE
Feature request: Timestamps in hooked browser left pane

### ISSUE THREAD

**Issue by Viss** [beefproject/beef#761]
It would be really handy to see at a glance the timestamps of when the browsers were hooked. When I use beef I employ it for weeks at a time, and timestamps are a valuable bit of intel. It would help workflow to not have to click every entry in the hooked browser list to get the timestamp

**Comment by bcoles** [comment]
Rather than try to cram data into the left panel, I think the best approach would be to have a separate 'Hooked Browsers' top-level tab (along side 'Getting Started' and 'Logs') which displays the online and offline hooked browsers in a sortable data grid table format, including the `first_seen` and `last_seen` properties.

This will probably require a change to one of the existing API endpoints, such as `/api/hooks?token=<token>`, to include `hb.firstseen` and `hb.lastseen`, or a new REST API endpoint.

This could be achieved with the following diff to the `get_hb_details` method in `core/main/rest/handlers/hookedbrowsers.rb`:

```
+              'firstseen' => hb.firstseen,
+              'lastseen' => hb.lastseen
```

**Comment by bcoles** [comment]
Hey @Viss 

This has finally been implemented, more than 6 years later, in #1674. We move fast around here.

Rather than implementing timestamps in the left pane, I've instead implemented a new 'Zombies' tab, which contains a sortable table of zombies, including the `firstseen` and `lastseen` properties, as described above. This would provide the exact information you want, except that these properties are formated as a string, making sorting useless. I expected this to be fixed some time in the next 6 years or so.

Also, retrieving the zombie details (including the `firstseen` and `lastseen` properties) has long been possible via the REST API, sort of. It kinda sucks.

The entire UI and REST interface need a rework. One step at a time...

Closing this issue.
